tests/reftests/run.ml

```diff
@@ -418,16 +418,17 @@ module Parse = struct
           | "|" :: "sed-cmd" :: cmd :: r ->
             let re =
               seq [
+                Re.str "+ ";
                 rep any;
                 alt [ set "/\\\"" ];
                 Re.str cmd;
                 opt @@ Re.str ".exe";
                 opt @@ char '"';
-                Re.str " \""
+                Re.str " "
```

Comment:
```suggestion
                Re.space
```

Reply:
Re.space does not have the same behaviour as `str " "` (the first one accepts other space characters that we should not accept)